scripts/max-loader.js

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env node
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var async = require('async');
+var fs = require('fs');
+var path = require('path');
+var request = require('request');
+var requestProgress = require('request-progress');
+var mkdirp = require('mkdirp');
+var cp = require('child_process');
+var CronJob = require('cron').CronJob;
+
+// set up mozlog, default is `heka`
+var mozlog = require('mozlog');
+mozlog.config({
+  app: 'fxa-geodb'
+});
+var log = mozlog();
+
+var target = path.join(process.cwd(), '../db');
```

Comment:
should be:

```js
path.join(process.cwd(), '..', 'db');
```
path.join takes care of inserting the appropriate directory separator for the environment.